core/src/types.ts

```diff
@@ -1,4 +1,5 @@
 import type {ReadableSignal, StoreOptions, SubscribableStore, WritableSignal} from '@amadeus-it-group/tansu';
+import type {FACTORY_WIDGET_NAME} from './config';
```

Comment:
I am not sure if we already had one without this change, but this for sure creates a circular dependency between our files, fortunately only for types, but it would still be better to avoid it.